### DISCUSSION TITLE
Wishes for Improvements

### DISCUSSION THREAD

**Discussion by Barmarel** [Tribler/tribler#6697] [Ideas]
**After using Tribler for a while, I have the following suggestions for improvements:**

1. The sequential download mode for the selected distribution is not enough.

2. Lacks the ability to specify the size of fonts interface.

3. Not enough to display information about already downloaded data in the form of MB.

4. Not enough a brief description of the application functions in the form of tips or a full FAQ built into the application. (This would make the app easier to use for beginners).

5. Missing a page with a description of the distribution (so that when you click on distribution the page will open with a cover and a description of the distribution). _You can see something similar in the MediaGet PC app._

6. There is not enough mobile client.

7. Also do not need the ability to dynamically mark only some files for download from the selected distribution. Exactly as it is done in usual torrent clients.

8. Also lacking is a system for evaluating handouts by users. So that you can see beforehand if there is something wrong with the giveaway.

9. It would also be cool if Tribler was in .AppImage format, if possible.

And another thing - In the screen shot of the application from the official site Tribler, handouts are displayed as a tile with covers, but in the application itself, handouts are displayed only as a list.

**Comment by devos50** [comment]
Thank you for using Tribler and your feedback!

> The sequential download mode for the selected distribution is not enough.

Could you specify what you mean by this? What kind of additional download mode behavior would you like to see?

> There is not enough mobile client.

We experimented a few years ago with a mobile Tribler client on Android. Unfortunately, this is challenging without major modifications since Tribler is entirely written in the Python programming language, and Android support for Python is poor. Nonetheless, it would be great to eventually ship an Android (and maybe even iOS) application 👍 .

> In the screen shot of the application from the official site Tribler, handouts are displayed as a tile with covers, but in the application itself, handouts are displayed only as a list.

Thanks for pointing this out. Because Tribler is continuously evolving, there are some inconsistencies with the screenshots on our website indeed.

**Reply by Barmarel** [reply]
>Could you specify what you mean by this? What kind of additional download mode behavior would you like to see?

The Torrent program downloads files in chunks without observing the sequence, as it chooses to download the parts that have a higher upload speed first. As a result, the download speed of the entire torrent is increased. This is good for games and programs, but if you download a movie and want to start watching it without waiting for the end of download (so that it continues to be downloaded in the background simultaneously with the playback), you should forcibly enable sequential downloading of its parts in the program settings. 

This option is available in all popular torrent clients.

---
I can also explain the other points in more detail.
Because I am writing through a translator, it may have distorted some of the information.

**Reply by NektoNektovich** [reply]
appimage is very necessary, because not everyone uses ubuntu and debian, and they can leave with deb! in addition, there is no tribler in the repositories! I will add that a forced file verification function is needed! :)

appimage очень нужен, поскольку не все пользуются ubuntu and debian, да и могут уйти с deb! к тому же триблера нет в репозиториях! добавлю, что нужна функция принудительной проверки файла! :)
